### PR TITLE
ci: update mergify rules for kubernetes 1.27

### PR DESCRIPTION
**Pull request by riya-singhal31**
Updated mergify rules to consider tests on Kubernetes 1.27

### CODE REVIEW

**Review by Madhu-1 — CHANGES_REQUESTED** [review]
changes are required here first https://github.com/ceph/ceph-csi/blob/devel/.github/workflows/pull-request-commentor.yaml

File: .github/workflows/pull-request-commentor.yaml [hunk 1]
```diff
@@ -44,6 +44,13 @@ jobs:
           body: |
             /test ci/centos/k8s-e2e-external-storage/1.26
 
+      - name: Add comment to trigger external storage tests for Kubernetes 1.27
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            /test ci/centos/k8s-e2e-external-storage/1.27
```
Comment (Madhu-1) [inline .github/workflows/pull-request-commentor.yaml]: Can we run all the tests on kubernetes 1.27 and see if deployment and E2E pass before mering the PR?

Ideally, we should have 2 PR's one for GitHub action change and another for mergify. If github action works fine, we can merge the mergigy PR as we won't be blocking other PRs from mergify if there is any issue with 1.27

Comment (riya-singhal31) [reply .github/workflows/pull-request-commentor.yaml]: Yes, sure. I'll run all the tests.

Okay, I'll add that to another PR then.
Thanks!

**Comment by riya-singhal31** [comment]
@Madhu-1 could you please remove ci/skip/e2e label here. Seems like I don't have access to do so.

**Comment by Madhu-1** [comment]
> @Madhu-1 could you please remove ci/skip/e2e label here. Seems like I don't have access to do so.

its added by the mergify bot, i think it will readd it again if i remove it can you please test this one on any other PR's?

**Comment by riya-singhal31** [comment]
> > @Madhu-1 could you please remove ci/skip/e2e label here. Seems like I don't have access to do so.
> 
> its added by the mergify bot, i think it will readd it again if i remove it can you please test this one on any other PR's?

Yes, sure.
I've added a separate PR for github actions as well.

**Comment by riya-singhal31** [comment]
1.27 is failing here, https://github.com/ceph/ceph-csi/pull/3735.
So, we can go ahead with merging the github action PR only for now.
@Madhu-1

**Comment by Madhu-1** [comment]
> 1.27 is failing here, #3735. So, we can go ahead with merging the github action PR only for now. @Madhu-1

No as the 1.27 CI is failing we cannot merge the PR.

**Review by Madhu-1 — CHANGES_REQUESTED** [review]
Not possible to merge this one until 1.2 CI passes

**Comment by Madhu-1** [comment]
@mergifyio rebase

**Comment by riya-singhal31** [comment]
1.27 CI is passing now, so should we merge this now? 
cc @Madhu-1 @nixpanic

**Review by nixpanic — APPROVED** [review]
Seems Kubernetes 1.27 CI jobs are passing in #3608.

**Comment by riya-singhal31** [comment]
> Seems Kubernetes 1.27 CI jobs are passing in #3608.

Yes!

**Comment by humblec** [comment]
considering 1.27 passes, @Madhu-1 can you remove the `requested changes` ?

**Review by Madhu-1 — APPROVED** [review]

**Comment by Madhu-1** [comment]
@mergifyio rebase

**Pull request merged**